docs/reference/sql/endpoints/odbc/applications.asciidoc

```diff
@@ -0,0 +1,32 @@
+[role="xpack"]
```

Comment:
This page should be merged inside the `Client Application` page - our docs shouldn't differentiate between ODBC and JDBC apps - they all connect to SQL just in a different way.